### PR TITLE
[WIP/DO NOT MERGE] Cosmetic changes for build steps

### PR DESCRIPTION
**Pull request by sa2ajj**
Main goal: unify parameter handling (so unknown parameters would cause errors).

### CODE REVIEW

File: master/buildbot/steps/master.py [hunk 1]
```diff
@@ -38,18 +38,18 @@ class MasterShellCommand(BuildStep):
     description = 'Running'
     descriptionDone = 'Ran'
     descriptionSuffix = None
-    renderables = ['command', 'env', 'workdir']
+    renderables = BuildStep.renderables + ['command', 'env']
```
Comment (jaredgrubb) [inline master/buildbot/steps/master.py]: I'm not 100% on this, but I dont think you have to pull in base-class renderables. I think it accumulates them across all bases automatically, right?

Comment (sa2ajj) [inline master/buildbot/steps/master.py]: I need to check whether it really accumulates them automatically.

In any case, a seemingly simple change caused a test case to error.

Comment (tardyp) [inline master/buildbot/steps/master.py]: Yes it does. See accumulate class list

Le ven. 16 janv. 2015 13:46, Mikhail Sobolev <notifications@github.com> a
écrit :

**Comment by djmitche** [comment]
Aside from the unnecessary `renderables` changes mentioned above, this looks good so far.

**Comment by sa2ajj** [comment]
Superceded by GH-1551

**Pull request closed without merging**